src/composables/useVotingEscrowLocks.ts

```diff
@@ -0,0 +1,111 @@
+import { computed, reactive } from 'vue';
+import useGraphQuery, { subgraphs } from '@/composables/queries/useGraphQuery';
+import useWeb3 from '@/services/web3/useWeb3';
+import useConfig from '@/composables/useConfig';
+import { bnum, isSameAddress } from '@/lib/utils';
+import useVeBalLockInfoQuery from '@/composables/queries/useVeBalLockInfoQuery';
+import useVotingGauges from '@/composables/useVotingGauges';
+import configs from '@/lib/config';
+import { networkId } from '@/composables/useNetwork';
+import useExpiredGaugesQuery from '@/composables/queries/useExpiredGaugesQuery';
+import { VotingGaugeWithVotes } from '@/services/balancer/gauges/gauge-controller.decorator';
+import useVeBal, { isVotingTimeLocked } from '@/composables/useVeBAL';
+
+/**
+ * TYPES
+ */
+export type VotingEscrowLock = {
+  votingEscrowID: {
+    id: string;
+  };
+  updatedAt: number;
+};
+
+type VotingEscrowLockQueryResponse = {
+  votingEscrowLocks: VotingEscrowLock[];
+};
+
+export default function useVotingEscrowLocks() {
+  /**
+   * COMPOSABLES
+   */
+  const { account } = useWeb3();
+  const { networkConfig } = useConfig();
+  const veBalLockInfoQuery = useVeBalLockInfoQuery();
+  const { votingGauges: allVotingGauges } = useVotingGauges();
+  const { veBalBalance } = useVeBal();
+
+  const votingEscrowLocksQueryEnabled = computed(() => !!account.value);
+  const votingEscrowLocksQuery = useGraphQuery<VotingEscrowLockQueryResponse>(
+    subgraphs.gauge,
+    ['votingEscrowLocks', veBalLockInfoQuery.data.value?.lockedAmount],
```

Comment:
The point of keeping query keys in a single constants file is so that we don't end up with clashes across the app when queries like this are created within other composables. Please move this query key into the constants file.